### ISSUE TITLE
+vterm/toggle & +vterm/here Stop Working After Opening =mu4e

### ISSUE THREAD

**Issue by jterk** [hlissner/doom-emacs#6116]
### What did you expect to happen?

vterm can be used before & after opening mu4e (either in a popup window, or in the current window)

### What actually happened?

Attempting to open a vterm window after starting mu4e causes the UI to flash as if something is happening, but vterm does not open.

The *Messages* buffer contains no error/warning indicating what might be happening.

### Describe your attempts to resolve the issue

Not using mu4e is the only thing that makes vterm work for me at the moment.

### Steps to reproduce

1. Configure the mu4e module
2. Open mu4e with `C-c M M`
3. Open vterm with `C-c o t` or `C-c o T`

### System Information

https://pastebin.com/jVbk8KAs

**Comment by jterk** [comment]
You can see my doom config here: https://github.com/jterk/dotfiles/tree/main/doom/.doom.d

mu4e is configured with:

```
(setq +mu4e-backend nil)
(set-email-account!
 "Employer"
 '((mu4e-sent-folder . "/Employer/[Gmail]/Sent Mail")
   (mu4e-drafts-folder . "/Employer/Drafts")
   (mu4e-trash-folder . "/Employer/Trash")
   (mu4e-refile-folder . "/Employer/[Gmail]/All Mail")
   (smtpmail-smtp-user . "jterk@employer.com")
   (user-mail-address . "jterk@employer.com")))

```

**Comment by jterk** [comment]
This was a problem with macOS 12.3 beta 3; fixed in 12.3 beta 4
